js/simple-stats.js

```diff
@@ -0,0 +1,47 @@
+const SimpleStats = function (flightLog) {
+    const frames = _(flightLog.getChunksInTimeRange(flightLog.getMinTime(), flightLog.getMaxTime()))
+        .map(chunk => chunk.frames).flatten().value(),
+        fields = _.map(flightLog.getMainFieldNames(), (f) => {
+            // fix typo. potential bug in either FW or BBE
+            if (f == "BaroAlt") { return "baroAlt"; } else { return f; }
```

Comment:
Please use strict equality https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality

Reply:
Thank you for the heads up! I'm going to include request from #309 too :ok_hand: